### PR TITLE
Remove files of remote dir before copying

### PR DESCRIPTION
**Pull request by ThomasRutzer**
I have added an optional parameter that, if set, will delete existing files in the remote folder before the copy operation. 
That's useful for deploying hashed files etc...

Maybe you can take a look at it and test it.

### CODE REVIEW

**Comment by Riley-Brown** [comment]
Any updates on this? @garygrossgarten 😁

File: src/index.ts [hunk 1]
```diff
@@ -133,6 +147,18 @@ async function putDirectory(
   }
 }
 
+async function cleanDirectory(ssh: node_ssh, remote: string, verbose = true) {
+  try {
+    await ssh.execCommand(`rm -rf ${remote}/*`);
```
Comment (gojanpaolo) [inline src/index.ts:152]: Unfortunately, this did not work with my windows server. I used `rmdir /s /q ${remote}` instead

Comment (ThomasRutzer) [reply src/index.ts:152]: cool! Would still be nice to have this PR reviewed / merged. Think it's a useful feature...

**Comment by garygrossgarten** [comment]
@ThomasRutzer  Sorry for being quiet for so long. Have been quiet busy lately and not using this action too much. But I merged your PR now and it's available in 0.7.0. Thank you so much!

**Pull request merged**